pages/common/p7zip.md

```diff
@@ -0,0 +1,24 @@
+# p7zip
+
+> Wrapper on 7-Zip file archiver with high compression ratio.
```

Comment:
Should it be `Wrapper of 7-Zip ...` instead?

Reply:
line 3 was copied from the man pages. but I will go ahead and make the changes.